client/my-sites/hosting/github/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { isEnabled } from '@automattic/calypso-config';
+import { useSelector } from 'react-redux';
+import { GithubAuthorizeCard } from 'calypso/my-sites/hosting/github/github-authorize-card/index';
+import { GithubConnectCard } from 'calypso/my-sites/hosting/github/github-connect-card/index';
```

Comment:
It's not idiomatic to include the `/index` suffix in the imports. Works though 🤷